core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -203,6 +207,10 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
       }
     }
 
+    private def isExecutorAlive(executorId: String): Boolean = synchronized {
```

Comment:
this is a super nit but I think we usually name things like "executorIsAlive" (a statement rather than a question)